### DISCUSSION TITLE
Welcome to DataMapper Discussions!

### DISCUSSION THREAD

**Discussion by turnerrainer** [buerokratt/DataMapper#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RaulAltmae** [comment]
@turnerrainer For Training-module delete theses helper functions must be defined 
```typescript
Handlebars.registerHelper('toJSON', function(obj) {
    return JSON.stringify(obj);
});
```
```typescript
Handlebars.registerHelper('eq', (a, b) => a == b)
```
